samples/system-test/vision_async_batch_annotate_images_beta.test.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Apologies for not being super clear.  For tests - I'm totally ok lumping those together into single files.  I don't feel super strongly one way or the other :)

Reply:
I think I will keep them in separate files. I think GAPIC might generate single test files too.

Reply:
The DLP canonicals have a separate test file per snippet FYI:
https://github.com/googleapis/nodejs-dlp/pull/154/files#diff-8bba442c891af76045c9128879591341